app/lib/bk/compat/parsers/gha/expressions.rb

```diff
@@ -143,6 +143,21 @@ def self.replace_context_secrets(var_name)
         "$$GITHUB_SECRET_#{var_name.upcase}"
       end
 
+      def self.replace_context_runner(var_name)
+        case var_name
+        when "name"
+          "$$BUILDKITE_AGENT_NAME"
+        when "os"
+          "$$BUILDKITE_AGENT_META_DATA_OS"
+        when "arch"
+          "$$BUILDKITE_AGENT_META_DATA_ARCHITECTURE"
+        when /(temp|tool_cache)/
+          "$$TMPDIR"
+        else
+          ""
```

Comment:
Previously it printed out a message stating it was not possible to translate. I'm not sure it is a good idea to just remove information that was previously present. How about doing something similar now?

```suggestion
          "There is no translation for github.#{var_name}"
```